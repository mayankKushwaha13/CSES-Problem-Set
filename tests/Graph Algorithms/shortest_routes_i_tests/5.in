10 20
8 9 3
2 3 8
10 5 3
2 5 3
5 6 6
3 4 3
10 3 8
4 5 5
4 9 10
6 7 7
5 9 5
4 1 7
8 6 7
4 8 5
1 2 8
5 4 7
10 8 9
7 8 9
9 10 10
9 6 7
