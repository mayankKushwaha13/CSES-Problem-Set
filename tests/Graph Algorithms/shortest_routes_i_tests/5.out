0 8 16 18 11 17 24 23 16 26 
